SU-HSDO/suhumsci
# SHS-5617: CSS Code Cleanup: Migrate SASS color variables to CSS variables

# READY FOR REVIEW

## Summary
- Migrate SASS color variables to CSS variables
- Remove color-pairings mixins

## Need Review By (Date)
05/31

## Urgency
medium

## Steps to Test
1. Run VRT
2. Confirm everything still looks the same
3. There are some false negatives, some in the order of the content and other ones in some buttons (check images below ). You can manually test this by changing the `pairing` classname:

![Screenshot 2024-05-29 at 5 43 51 PM](https://github.com/SU-HSDO/suhumsci/assets/48533432/2e3293b7-80c6-437e-ba61-d780b2035805)

![Screenshot 2024-05-29 at 5 43 45 PM](https://github.com/SU-HSDO/suhumsci/assets/48533432/f3f1542d-7dd0-4872-8535-ac4cf1c973a1)

![Screenshot 2024-05-29 at 5 43 58 PM](https://github.com/SU-HSDO/suhumsci/assets/48533432/dd8c83a4-80a1-4658-8535-ee9e58b46970)



## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
